Index: ponto_online_app/all_routes.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>from ponto_online_app.routes import index\r\nfrom ponto_online_app.routes import login\r\nfrom ponto_online_app.routes import auth\r\nfrom ponto_online_app.routes import logout\r\nfrom ponto_online_app.routes import cadastro\r\nfrom ponto_online_app.routes import validation_point\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/ponto_online_app/all_routes.py b/ponto_online_app/all_routes.py
--- a/ponto_online_app/all_routes.py	(revision 3d5d6464523dec52705dc97eccefe873c1589397)
+++ b/ponto_online_app/all_routes.py	(date 1662679783305)
@@ -4,3 +4,4 @@
 from ponto_online_app.routes import logout
 from ponto_online_app.routes import cadastro
 from ponto_online_app.routes import validation_point
+from ponto_online_app.services.dados import teste
